backend/src/controllers/task.js

```diff
@@ -25,16 +25,8 @@ router.get('/tasks', async (req, res) => {
 
   // DELETE task
 router.delete('/tasks/:id', async (req, res) => {
-  console.log(req.params.id)
   try {
-    const task = await Task.findById(req.params.id);
-
-    if (!task) {
-      return res.status(404).json({ message: 'Task not found' });
-    }
-
-    await task.remove();
-
+    await Task.findByIdAndRemove(req.params.id);
```

Comment:
What you wrote there for Task deletion is right, but I wanted to show you that you can achieve almost the same with only one line, thanks to the `findByIdAndRemove()` method.